Add reference to service key. (#1694)
cloudflare/cloudflare-docs: products/network-interconnect/src/content/partners/megaport.md

## Create a VXC with a service key

> Once your Cloudflare account team creates the service key and shares with you, anybody with the key can create a connection.

Reviewer: ```suggestion
After the Cloudflare account team creates the service key and shares it with you, anybody with the key can create a connection.
```

Reviewer: Small suggestion that I think will make this a little clearer for users

> Once your Cloudflare account team creates the service key and shares with you, anybody with the key can create a connection.

Reviewer: ```suggestion
The Cloudflare account team creates the Service Key and sends it to you via email. After the Service Key is sent, anyone who has the key can use it to create a connection.
```

1. Log in to the [Megaport Portal](https://portal.megaport.com/).
1. Find the Megaport you want to use for the connection and click **+Connection**.

> 1. Under **Choose Destination Type**, click **Enter Service Key**. Your Cloudflare account team will provide the Service Key via email.

Reviewer: I deleted my previous suggestion after thinking about it some more. I think it makes sense to add the line about receiving the service key via email to the sentence before the procedure.

> 1. Under **Choose Destination Type**, click **Enter Service Key**. Your Cloudflare account team will provide the Service Key via email.

Reviewer: ```suggestion
1. Under **Choose Destination Type**, click **Enter Service Key**.
```
